### ISSUE TITLE
The 'rxjs-no-unsafe-scope' rule threw an error

### ISSUE THREAD

**Issue by Moeriki** [cartant/rxjs-tslint-rules#61]
```
The 'rxjs-no-unsafe-scope' rule threw an error in './src/app/store/effects.ts':
TypeError: Cannot read property 'kind' of undefined
    at Object.isParameterDeclaration (./node_modules/rxjs-tslint-rules/node_modules/tsutils/typeguard/2.8/node.js:514:17)
    at Walker.isUnsafe (./node_modules/rxjs-tslint-rules/dist/rules/rxjsNoUnsafeScopeRule.js:84:45)
    at Walker.visitNode (./node_modules/rxjs-tslint-rules/dist/rules/rxjsNoUnsafeScopeRule.js:61:38)
    at ./node_modules/tslint/lib/language/walker/syntaxWalker.js:535:63
    at visitNode (./node_modules/typescript/lib/typescript.js:15779:24)
    at Object.forEachChild (./node_modules/typescript/lib/typescript.js:16009:21)
    at Walker.SyntaxWalker.walkChildren (./node_modules/tslint/lib/language/walker/syntaxWalker.js:535:12)
    at Walker.SyntaxWalker.visitConditionalExpression (./node_modules/tslint/lib/language/walker/syntaxWalker.js:72:14)
    at Walker.SyntaxWalker.visitNode (./node_modules/tslint/lib/language/walker/syntaxWalker.js:323:22)
    at Walker.visitNode (./node_modules/rxjs-tslint-rules/dist/rules/rxjsNoUnsafeScopeRule.js:65:36)
```

_./src/app/store/effects.ts_

```js
/*  1 */ import { Inject, Injectable, LOCALE_ID } from '@angular/core';
/*  2 */ import { Actions, Effect, ofType } from '@ngrx/effects';
/*  3 */ import { Action } from '@ngrx/store';
/*  4 */ import { Observable, of } from 'rxjs';
/*  5 */ import { catchError, map, switchMap } from 'rxjs/operators';
/*  6 */ 
/*  7 */ import { CmLocale } from 'app/cm-common';
/*  8 */ 
/*  9 */ import { CmApiLabelService } from '../service';
/* 10 */ import * as featureActions from './actions';
/* 11 */ 
/* 12 */ @Injectable()
/* 13 */ export class AppStoreEffects {
/* 14 */   constructor(
/* 15 */     @Inject(LOCALE_ID) private localeId: CmLocale,
/* 16 */     private actions$: Actions,
/* 17 */     private labelService: CmApiLabelService,
/* 18 */   ) {}
/* 19 */ 
/* 20 */   @Effect()
/* 21 */   loadRequestEffect$: Observable<Action> = this.actions$.pipe(
/* 22 */     ofType<featureActions.LoadRequestAction>(
/* 23 */       featureActions.ActionTypes.LOAD_REQUEST,
/* 24 */     ),
/* 25 */     switchMap(({ payload: { applyLabelIdAsPrefix, labelId, localeId } }) =>
/* 26 */       this.labelService.getLabels(labelId, localeId || this.localeId).pipe(
/* 27 */         map(
/* 28 */           (labels) =>
/* 29 */             new featureActions.LoadSuccessAction({
/* 30 */               labels,
/* 31 */               prefix: applyLabelIdAsPrefix ? labelId : undefined,
/* 32 */             }),
/* 33 */         ),
/* 34 */         catchError((error) =>
/* 35 */           of(new featureActions.LoadFailureAction({ error })),
/* 36 */         ),
/* 37 */       ),
/* 38 */     ),
/* 39 */   );
/* 40 */ }
```

**Comment by cartant** [comment]
Thanks for including the source that sees the error effected. I'll add it as a test to try to reproduce the error, later today.

**Comment by cartant** [comment]
The test I added effects an error - which I will fix - but it's not the one you've mentioned in the issue.

```
The 'rxjs-no-unsafe-scope' rule threw an error in '/Users/nicholasjamieson/Documents/Projects/other/rxjs-tslint-rules/temp/fixture.ts':
TypeError: Cannot read property '0' of undefined
    at Walker.isUnsafe (/Users/nicholasjamieson/Documents/Projects/other/rxjs-tslint-rules/build/rules/rxjsNoUnsafeScopeRule.js:83:51)
    at Walker.visitNode (/Users/nicholasjamieson/Documents/Projects/other/rxjs-tslint-rules/build/rules/rxjsNoUnsafeScopeRule.js:61:38)
    at /Users/nicholasjamieson/Documents/Projects/other/rxjs-tslint-rules/node_modules/tslint/lib/language/walker/syntaxWalker.js:535:63
    at visitNode (/Users/nicholasjamieson/Documents/Projects/other/rxjs-tslint-rules/node_modules/typescript/lib/typescript.js:15779:24)
    at Object.forEachChild (/Users/nicholasjamieson/Documents/Projects/other/rxjs-tslint-rules/node_modules/typescript/lib/typescript.js:15878:21)
    at Walker.SyntaxWalker.walkChildren (/Users/nicholasjamieson/Documents/Projects/other/rxjs-tslint-rules/node_modules/tslint/lib/language/walker/syntaxWalker.js:535:12)
    at Walker.SyntaxWalker.visitBindingElement (/Users/nicholasjamieson/Documents/Projects/other/rxjs-tslint-rules/node_modules/tslint/lib/language/walker/syntaxWalker.js:42:14)
    at Walker.SyntaxWalker.visitNode (/Users/nicholasjamieson/Documents/Projects/other/rxjs-tslint-rules/node_modules/tslint/lib/language/walker/syntaxWalker.js:296:22)
    at Walker.visitNode (/Users/nicholasjamieson/Documents/Projects/other/rxjs-tslint-rules/build/rules/rxjsNoUnsafeScopeRule.js:65:36)
    at /Users/nicholasjamieson/Documents/Projects/other/rxjs-tslint-rules/node_modules/tslint/lib/language/walker/syntaxWalker.js:535:63
```

**Comment by cartant** [comment]
This should be fixed in 4.11.1.

**Comment by Moeriki** [comment]
Fix confirmed.
